back/package.yaml

```diff
@@ -197,3 +200,17 @@ tests:
     - temporary
     - webdriver
     - yaml
+
+benchmarks:
```

Comment:
You also should add `--enable-benchmarks` (or `--bench --no-run-benchmarks` or something, look at `stack --help`) in the Travis config so that they would be compiled during CI.